gradio/routes.py

```diff
@@ -670,6 +670,8 @@ async def queue_join(
             request: fastapi.Request,
             username: str = Depends(get_current_user),
         ):
+            blocks = app.get_blocks()
```

Comment:
we need to use get_blocks, because the blocks reference passed from create_app does not get refreshed on reloaded